### PR TITLE
Creating ContactBundle with two pages

### PR DESCRIPTION
**Pull request by LisKorzun**

### CODE REVIEW

File: src/TestLab/ContactBundle/Controller/IndexController.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+/**
+ * @Route(service="testlab.contact.controller.index")
+ */
+class IndexController extends AbstractController
+{
+    /**
+     * @Route("/create", name="test_lab_contact_createpage")
+     * @Template
+     */
+    public function createAction()
```
Comment (EduardTrayan) [inline src/TestLab/ContactBundle/Controller/IndexController.php]: Please take a look at http://symfony.com/doc/current/cookbook/templating/render_without_controller.html.
Seems, you don't need to use your custom controllers just for rendering templates - please use `template` controller instead

File: src/TestLab/ContactBundle/Controller/IndexController.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    {
+        return [];
+    }
+
+    /**
+     * @Route("/view", name="test_lab_contact_viewpage")
+     * @Template
+     */
+    public function viewAction()
```
Comment (EduardTrayan) [inline src/TestLab/ContactBundle/Controller/IndexController.php]: See above

File: src/TestLab/ContactBundle/Resources/config/routing.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+test_lab_contact_createpage:
+    resource: "@TestLabContactBundle/Controller/"
+    type:     annotation
+
+test_lab_contact_viewpage:
+    resource: "@TestLabContactBundle/Controller/"
```
Comment (EduardTrayan) [inline src/TestLab/ContactBundle/Resources/config/routing.yml]: Why do you import all routes twice?

File: app/AppKernel.php [hunk 4]
```diff
@@ -18,6 +18,7 @@ public function registerBundles()
             new Sensio\Bundle\FrameworkExtraBundle\SensioFrameworkExtraBundle(),
             new TestLab\CommonBundle\TestLabCommonBundle(),
             new TestLab\WebsiteBundle\TestLabWebsiteBundle(),
+            new TestLab\ContactBundle\TestLabContactBundle(),
```
Comment (EduardTrayan) [inline app/AppKernel.php]: Please remove this bundle and move all files to website bundle

File: app/Resources/views/base.html.twig [hunk 5]
```diff
@@ -1,13 +1,33 @@
...
-    </head>
-    <body>
-        {% block body %}{% endblock %}
-        {% block javascripts %}{% endblock %}
-    </body>
+<head>
+    <meta charset="UTF-8" />
+    <title>{% block title %}Welcome{% endblock %}</title>
+    {% block stylesheets %}
+        <link rel="stylesheet" href="{{ asset('bundles/testlabcontact/css/bootstrap.css') }}" />
```
Comment (EduardTrayan) [inline app/Resources/views/base.html.twig]: Please, for the first, use named assets http://symfony.com/doc/current/cookbook/assetic/asset_management.html#using-named-assets
for the second, 
```
flowplayer_js:
            inputs:
                - @EpamVideoportalWebsiteBundle/Resources/public/flowplayer/js/flowplayer-3.2.13.min.js
                - @EpamVideoportalWebsiteBundle/Resources/public/flowplayer/js/flowplayer.playlist-3.2.11.min.js
            output: js/flowplayer.js

<script src="{{asset('js/flowplayer.js')}}"></script>
```

File: src/TestLab/ContactBundle/Controller/IndexController.php (hunk 1 above)
Comment (LisKorzun) [inline src/TestLab/ContactBundle/Controller/IndexController.php]: ок

File: src/TestLab/ContactBundle/Controller/IndexController.php (hunk 2 above)
Comment (LisKorzun) [inline src/TestLab/ContactBundle/Controller/IndexController.php]: ок

File: src/TestLab/ContactBundle/Resources/config/routing.yml (hunk 3 above)
Comment (LisKorzun) [inline src/TestLab/ContactBundle/Resources/config/routing.yml]: ок

File: app/AppKernel.php (hunk 4 above)
Comment (LisKorzun) [inline app/AppKernel.php]: ок

File: app/Resources/views/base.html.twig (hunk 5 above)
Comment (LisKorzun) [inline app/Resources/views/base.html.twig]: ок

File: app/Resources/views/base.html.twig [hunk 6]
```diff
@@ -1,13 +1,32 @@
...
+        <link rel="stylesheet" href="{{ asset('css/style.css') }}" />
+    {% endblock %}
+    <link rel="icon" type="image/x-icon" href="{{ asset('favicon.ico') }}" />
+</head>
+<body>
+<nav class="navbar navbar-default navbar-static-top">
+    <div class="container">
+        <div class="navbar-header">
+            <a  href="http://www.it-academy.by/" target="_blank">
+                <img class="logo" alt="Brand" src="{{ asset('bundles/testlabwebsite/images/logo.png') }}">
```
Comment (EduardTrayan) [inline app/Resources/views/base.html.twig]: You can as well do it for images

File: app/config/config.yml [hunk 7]
```diff
@@ -36,6 +36,12 @@ assetic:
     use_controller: false
     bundles:        [ ]
     #java: /usr/bin/java
+    assets:
```
Comment (EduardTrayan) [inline app/config/config.yml]: :+1:

**Pull request merged**